### PR TITLE
2.9.0 release notes and highlights and Logstash volume docs

### PR DESCRIPTION
**Pull request by pebrc**
~I am going to open a separate PR for the highlights and add some documentation for the Logstash volume support.~

Actually I am going to push the highlights and Logstash docs to this branch as well, because of the links.

Preview https://cloud-on-k8s_7003.docs-preview.app.elstc.co/diff

### CODE REVIEW

File: docs/release-notes/2.9.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+
+
+[[enhancement-2.9.0]]
+[float]
+=== Enhancements
+
+* Add Support for volumeClaimTemplates for Logstash controller {pull}6884[#6884]
+* Add back runAsNonRoot=true to Beats >= 8.8.x {pull}6793[#6793]
```
Comment (thbkrkr) [inline docs/release-notes/2.9.0.asciidoc:15]:
```suggestion
* Enable runAsNonRoot=true for Beats >= 8.8.x {pull}6793[#6793]
```

File: docs/release-notes/2.9.0.asciidoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+[[bug-2.9.0]]
+[float]
+=== Bug fixes
+
+* Validating policy ID only when running in fleet mode for Elastic Agent {pull}6938[#6938] (issue: {issue}6903[#6903])
```
Comment (thbkrkr) [inline docs/release-notes/2.9.0.asciidoc:21]:
```suggestion
* Validating policy ID only when running in Fleet mode for Elastic Agent {pull}6938[#6938] (issue: {issue}6903[#6903])
```

File: docs/release-notes/2.9.0.asciidoc [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+* Add Selector to Logstash status {pull}6854[#6854]
+
+[[docs-2.9.0]]
+[float]
+=== Documentation improvements
+
+* Document Logstash connection to external Elasticsearch {pull}6895[#6895]
+* Fix rendering of autopilot recipes README {pull}6856[#6856]
```
Comment (thbkrkr) [inline docs/release-notes/2.9.0.asciidoc:29]: Exclude this one? 🤷‍♂️ 
```suggestion
```

File: docs/orchestrating-elastic-stack-applications/logstash.asciidoc [hunk 4]
```diff
@@ -284,6 +284,65 @@ spec:
...
+
+[source,yaml,subs="attributes,+macros,callouts"]
+----
+apiVersion: logstash.k8s.elastic.co/v1alpha1
+kind: Logstash
+metadata:
+  name: logstash
+spec:
+  count: 1
+  version: {version}
```
Comment (thbkrkr) [inline docs/orchestrating-elastic-stack-applications/logstash.asciidoc:321]:
```suggestion
  version: {version}
  config:
    queue.type: persisted
```

File: docs/orchestrating-elastic-stack-applications/logstash.asciidoc [hunk 5]
```diff
@@ -284,6 +284,65 @@ spec:
...
+        resources:
+          requests:
+            storage: 10Gi
+  config: 
+    log.level: info
+    queue.type: persisted
+    path.queue: /usr/share/logstash/pq
+----
+
+NOTE: The current implementation does not allow resizing of volumes even if your chosen storage class would support it. To resize a volume you have to fully delete the Logstash object, delete and recreate or resize the volume and create a new Logstash object. Also note that volume claims will not be deleted when you delete the Logstash object and have to be deleted manually. This behaviour might change in future versions of the ECK operator.
```
Comment (robbavey) [inline docs/orchestrating-elastic-stack-applications/logstash.asciidoc:345]: We should also mention that this is a breaking change to existing Logstash deployments, as this requires a change to the `StatefulSet` associated with Logstash

**Review by robbavey — APPROVED** [review]
LGTM

**Comment by robbavey** [comment]
@pebrc Would you mind if I put up a follow-up on this with a more comprehensive version of the Logstash docs, which adds some more detail to the use cases on PQ and DLQ including details on draining queues, etc?

**Comment by pebrc** [comment]
> 



> @pebrc Would you mind if I put up a follow-up on this with a more comprehensive version of the Logstash docs, which adds some more detail to the use cases on PQ and DLQ including details on draining queues, etc?

Sounds good! Let me merge this first (and back port)

**Comment by pebrc** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|2.9|[](https://github.com/elastic/cloud-on-k8s/pull/7021)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**